jupyterhub/tasks/main.yml

```diff
@@ -8,53 +8,42 @@
 
 
 - name: Install Python 3, Node, and NPM
-  apt: name={{item}} state=present update_cache=yes
+  apt: 
+     name: ['python3', 'python3-setuptools', 'python3-dev', 'python3-pip', 'sudo', 'npm', 'nodejs-legacy', 'git']
+     state: present 
+     update_cache: yes
   when: ansible_os_family == 'Debian'
-  with_items:
-  - python3
-  - python3-setuptools
-  - python3-dev
-  - python3-pip
-  - sudo
-  - npm
-  - nodejs-legacy
-  - git
   tags: jupyter-deps
 
 - name: Install Python 3, Node, and NPM
-  yum: name={{item}} state=present update_cache=yes
-  when: ansible_os_family == 'RedHat'
-  with_items:
-  - python36u
-  - python36u-setuptools
-  - python36u-devel
-  - python36u-pip
-  - sudo
-  - npm
-  - nodejs
-  - git
+  apt:
+     name: ['python37', 'python37-setuptools', 'python37-devel', 'python37-pip', 'sudo', 'npm', 'nodejs', 'git']
```

Comment:
Why did this change from yum to apt?

Reply:
Testing and I forgot to change it back.  Fixed.